startup/95-plans.py

```diff
@@ -120,29 +120,11 @@ def inner():
         yield from bps.abs_set(mono1.start_trajectory, '1', wait=True)
 
         # this should be replaced by a status object
+        # Updated below method. Should we remove the comment above?
```

Comment:
Yes, please remove the comment above and this comment ;)

Reply:
Ok. I can do that in the new branch that I created. And then do another PR. Then what should I do about this PR? What does the "Resolve conversation" button do?

Reply:
No, just make changes to this exact branch and push them to the PR - the changes will be reflected immediately. No need for a new PR. I guess, 'resolve conversation' means we all are satisfied with the changes, but may be wrong, I never used it.

Reply:
Just did the "git push" after making the changes

Reply:
I did not commit. Do I have to? Going to check git for updates for PR 5.

Reply:
I mean PR #6

Reply:
Push will do nothing before you commit. Sure, please commit first.

Reply:
Done.

Reply:
OK, this looks good now.